### ISSUE TITLE
Issue Pantheon Create | PHP Notice:  Undefined offset | kbox 0.9.18

### ISSUE THREAD

**Issue by magbicaleman** [kalabox/kalabox#479]
Every time I try to create an kbox application from Pantheon, I'm getting the following error from the build, I've tried ```kbox rebuild ``` many times, and ``` kbox uninstall ``` and starting the process all over, but none of it seems to get passed the error. 

``` Error while installing app: "myApp": EVENTS =>  Error dispatching event listeners [post-install].
VError: Error running ad hoc container with kalabox/terminus:v0.9.0.: PHP Notice:  Undefined offset: 0 in phar:///usr/local/bin/terminus/php/class-terminus-command.php on line 307
PHP Notice:  Undefined offset: 0 in phar:///usr/local/bin/terminus/php/class-terminus-command.php on line 317 ```

    at Object.<anonymous> (/usr/local/lib/node_modules/kalabox/node_modules/kalabox-engine-docker/docker.js:696:13)
    at processImmediate [as _immediateCallback] (timers.js:358:17) ```

**Comment by labboy0276** [comment]
Hello

This issue has been addressed and fixed in an upcoming version #424 

In the interim run this on the app:

```
kbox stop && kbox uninstall && kbox install
```

**Comment by magbicaleman** [comment]
@labboy0276 Thank you very much for your reply. I ran the command as you pasted it, and then ran them individuals but no cigar. Same issue. I would think ``` kbox rebuild ``` would do the stop, uninstall, install.

**Comment by pirog** [comment]
I am pretty sure this issue has been resolved in the latest 0.10.0 release. I would recommend you uninstall and reinstall with that version and see if the issue persists. 

If it does feel free to reopen.
